### PR TITLE
Add config for PHP CLI

### PR DESCRIPTION
**Pull request by akovalyov**
Currently PHP CLI configuration is not handled by Trellis, this PR aims to fix it. 

Usecase: emails, which are being processed via cronjobs are getting lost as they are being sent from background tasks, where default sendmail is configured instead of ssmtp.

Web:
image
CLI:
image

### CODE REVIEW

**Review by swalkinshaw — APPROVED** [review]
Good idea, thanks 🎉

File: roles/php/tasks/main.yml [hunk 1]
```diff
@@ -45,8 +45,13 @@
...
+- name: Copy PHP-FPM configuration file
   template:
-    src: php.ini.j2
+    src: php-fpm.ini.j2
     dest: /etc/php/7.4/fpm/php.ini
   notify: reload php-fpm
+
+- name: Copy PHP CLI configuration file
+  template:
+    src: php-fpm.ini.j2
```
Comment (TangRufus) [inline roles/php/tasks/main.yml:56]: Should this be?

```diff
-     src: php-fpm.ini.j2 
+     src: php-cli.ini.j2 
```

Comment (akovalyov) [reply roles/php/tasks/main.yml:56]: @TangRufus you're right. I've missed it. Should I submit another PR to fix that? /cc @swalkinshaw ?

Comment (TangRufus) [reply roles/php/tasks/main.yml:56]: Yes, please

Comment (akovalyov) [reply roles/php/tasks/main.yml:56]: @TangRufus done, https://github.com/roots/trellis/pull/1262

**Pull request merged**